spicy/toolchain/include/ast/types/unit.h

```diff
@@ -114,7 +114,7 @@ class Unit : detail::AssignIndices,
     }
 
     /**
-     * Returns all properties of a given name.
+      Returns all properties of a given name.
```

Comment:
This is now formatted inconsistently (no leading `* ` on lines in block). Let's either keep the old format or make this a single-line comment.